src/pages/_app.js

```diff
@@ -31,6 +32,10 @@ const G_TAG_KEY = process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS
 export default function App({ Component, pageProps }) {
   useGoogleTagsManager(G_TAG_KEY)
 
+  const [applyTabIndex, setApplyTabIndex] = useState(false)
```

Comment:
Seems like this `useState` is not needed here and could be moved directly into the Nav component. This would remove the props being passed all together. If there is a need to have it at this level create a custom hook and share it via imports instead of props.

Reply:
Moving it to the Nav worked!